sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/AggregationQuerySuite.scala

```diff
@@ -545,19 +576,21 @@ abstract class AggregationQuerySuite extends QueryTest with SQLTestUtils with Te
           |  count(distinct value2),
           |  sum(distinct value2),
           |  count(distinct value1, value2),
+          |  longProductSum(distinct value1, value2),
```

Comment:
yeah, right. I was thinking about just the inputs of this particular distinct agg function (records with other gid values are ignored).